packages/app/playwright/commons/gas.ts

```diff
@@ -5,7 +5,7 @@ import type { Provider, BN } from 'fuels';
 export const getGasConfig = async (provider: Provider) => {
   const chain = await provider.getChain();
   const nodeInfo = await provider.fetchNode();
-  const gasLimit = chain.consensusParameters.maxGasPerTx;
+  const gasLimit = chain.consensusParameters.maxGasPerTx.div(2);
```

Comment:
why are we dividing by 2?

Reply:
The current state of the fuel-core consider max gas limit, the gas for the entire transaction. But the gasLimit config is used only for script, because of that we need to set the gasLimit in a value that is smaller them the max with room for other costs. This is why we divide by 2.